core/src/main/java/com/valtech/aapm/restrictions/HasPropertyValuesPattern.java

```diff
@@ -45,32 +45,31 @@ public class HasPropertyValuesPattern implements RestrictionPattern {
     private final String propertyType;
     private static final String DENY = "deny";
 
-    List<String> operatorList = List.of("==", ">=", "<=","<",">");
-    private String operator = "==";
+    List<String> operators = List.of("DOUBLE_EQUALS", "GREATER_THAN_EQUALS", "LESS_THAN_EQUALS", "GREATER_THEN", "LESS_THEN");
```

Comment:
I would replace "DOUBLE_EQUALS" with "EQUALS" as this is more related to the meaning of the comparison.

Reply:
Fixed